src/index.ts

```diff
@@ -1,24 +1,16 @@
-// @flow
-import type { Stream, Sink, ScheduledTask, Time, Scheduler, Disposable } from '@most/types'
+import { Stream, Sink, ScheduledTask, Time, Scheduler, Disposable } from '@most/types' // eslint-disable-line no-unused-vars
```

Comment:
Can you help me understand why this `eslint-disable-line` is necessary?

Reply:
Otherwise `standardjs` set of eslint rules together with typescript parser think that these types are imported into the module but not used. Standardjs does not support overloading of `.eslintrc` so the only option I've found (besides replacing standardjs with plain eslint) was to add this comment.

Reply:
:+1: